classes/BadpwFailedLoginsDAO.inc.php

```diff
@@ -61,19 +61,19 @@ public function getByUsername(string $username) : ?BadpwFailedLogins {
 			SELECT *
 			FROM badpw_failedlogins
 			WHERE username = ?
-		', $username);
-		$row = $result->GetRowAssoc(false);
-		if ($result->RowCount()) {
-			return new BadpwFailedLogins($row['username'], $row['count'], strtotime($row['failed_login_time']));
-		}
-		/** @var UserDao */
-		$userDao = DAORegistry::getDAO('UserDAO');
-		$user = $userDao->getByUsername($username);
-		if ($user) {
-			$badpwObj = new BadpwFailedLogins($username, 0, time());
-			$this->_insertObject($badpwObj);
-			return $badpwObj;
+		', [(string)$username]);
+                
+                $row = (array)$result->current();
+
+                //User already in database
```

Comment:
Watch for mixing tabs with spaces.  For 3.3 and later, PKP is converting from tabs to spaces, PSR12 style:
https://github.com/pkp/pkp-lib/issues/5678

We should make similar changes to our plugins, but these should be a single commit which only changes tabs to spaces.